.github/workflows/release.yml

```diff
@@ -31,3 +31,5 @@ jobs:
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           GPG_SECRET: ${{ secrets.GPG_SECRET }}
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+          PLUGIN_PORTAL_KEY: ${{ secrets.PLUGIN_PORTAL_KEY }}
```

Comment:
Am I looking at this weird? I am trying a review from the new mobile app. Is the indentation off here?

Reply:
indentation seems correct, maybe it something with the mobile app